indigo_app/templates/indigo_api/work_form.html

```diff
@@ -85,6 +85,16 @@ <h5 class="card-header">Work details</h5>
           </div>
         </div>
 
+        <div class="form-row">
+          <div class="form-group col-md-6 offset-md-6">
+            <div class="form-check">
+              <input type="checkbox" class="form-check-input" id="{{ form.stub.id_for_label }}" name="{{ form.stub.html_name }}" value="on" {% if form.stub.value %}checked{% endif %}>
+              <label class="mb-0" for="{{ form.stub.id_for_label }}">Stub - this work will have no content</label>
+              <p class="form-text text-muted">Check this option for commencing or amending works that are not captured in detail.</p>
+            </div>
+          </div>
+        </div>
+
```

Comment:
just a note that we may want to be more general here in the future depending on what we encounter. for now we can assume that repealing works aren't generally stubs and that we won't be creating works for notices or similar if we aren't also importing them

Reply:
Can you suggest different wording that captures these?

We can indeed change later if necessary.

Reply:
"Check this option for works that are not captured in detail, e.g. commencing or amending works."?